packages/rancher-backup/charts/app-readme.md

```diff
@@ -0,0 +1,15 @@
+# Rancher Backup
+
+This chart enables ability to capture backups of the Rancher application and restore from these backups. This chart can be used to [migrate Rancher from one Kubernetes cluster to a different Kubernetes cluster](https://rancher.com/docs/rancher/v2.x/en/backups/v2.5/migrating-rancher/).
+
+For more information on how to use the feature, refer to our [docs](https://rancher.com/docs/rancher/v2.x/en/backups/v2.5/).
+
+This chart installs the following component:
```

Comment:
nit (assuming Backup and Restore are also being considered as chart components along with the operator workload)
```suggestion
This chart installs the following components:
```